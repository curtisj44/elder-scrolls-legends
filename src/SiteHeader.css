.site-header {
  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.5);
  left: 0;
  position: fixed;
  right: 0;
  top: 0;
  z-index: 100;
}

.site-header__heading {
  background: var(--dark-moon);
  font: 1rem/1 var(--serif);
  letter-spacing: 0.07em;
  padding: 0.6rem 16px;
  text-align: center;
  text-transform: uppercase;
}
