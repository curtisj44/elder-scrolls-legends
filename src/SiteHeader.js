// Dependencies
// ------------

// Libraries
import React from 'react';
// Styles
import './SiteHeader.css';

// Exports
// -------

export default () => {
  return (
    <header className="site-header" role="banner">
      <h1 className="site-header__heading">
        The Elder Scrolls: Legends
      </h1>
    </header>
  );
};
