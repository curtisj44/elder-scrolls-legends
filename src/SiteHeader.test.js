// Dependencies
// ------------

// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import SiteHeader from './SiteHeader';

// Setup
// -----

const component = renderer.create(<SiteHeader />);

// Tests
// -----

describe('SiteHeader', () => {
  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
