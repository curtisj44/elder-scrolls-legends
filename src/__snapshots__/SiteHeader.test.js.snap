// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader renders correctly 1`] = `
<header
  className="site-header"
  role="banner"
>
  <h1
    className="site-header__heading"
  >
    The Elder Scrolls: Legends
  </h1>
</header>
`;
